kubejs/server_scripts/tfc/recipes.js

```diff
@@ -241,4 +241,52 @@ const registerTFCRecipes = (event) => {
 		.itemOutputs('tfc:wrought_iron_grill')
 		.duration(60)
 		.EUt(8)
+
+	//Jigabit's stupid acetic acid to vinegar conversion
+
+		event.recipes.gtceu.chemical_reactor('vinegar_from_acetic_acid')
+        		.inputFluids(
+                            Fluid.of('minecraft:water', 950),
+                            Fluid.of('gtceu:acetic_acid', 50)
+                )
+        		.outputFluids(
+                			Fluid.of('tfc:vinegar', 1000)
+                )
+			    .circuit(1)
+        		.duration(30)
+        		.EUt(GTValues.VA[GTValues.HV])
```

Comment:
I think it's safe to make this MV like the distillery recipe? I understand MV is to not make the tfc recipe irrelevant 

also idk with chem reactor or mixer is more appropriate, now that I think of it, since you're just diluting the acid and not reacting anything